datamade/committee-oversight
# DOCKERIZE

## Overview

Closes #69. This PR containerizes the app for local development and updates the README with the new workflow. 

This blog post was very helpful in figuring out how to run `pg_restore` in a Docker container: https://simkimsia.com/how-to-restore-database-dumps-for-postgres-in-docker-container/

I also downloaded the Atom Dockerfile highlighting plugin: https://atom.io/packages/language-docker

Note: for now, Wagtail content backups and `hearings` database backups are completely separate. We may want to reconsider that in the future, especially once #63 is running for `hearings`, but that's outside the scope of this PR.

## Testing Instructions

* Follow instructions in the ["Running the app locally" section of the README](https://github.com/datamade/committee-oversight/tree/issue-069#running-the-app-locally) exactly to get the app running locally. Let's see if it works as promised!
* If the above goes smoothly, restore the Wagtail content by running: 

	```bash
	docker exec committeeoversight python manage.py load_cms_content
	```
* Navigate to http://localhost:8000/cms/images/. If it worked, you should see a second cow.
